packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -125,7 +125,7 @@ class RealAccountSignup extends Component {
                 <Localize
                     i18n_default_text='<0>You have added a Deriv {{currency}} account.</0><0>Make a deposit now to start trading.</0>'
                     values={{
-                        currency: CurrencyUtils.getCurrencyDisplayCode(currency),
+                        currency: CurrencyUtils.getCurrencyDisplayCode(currency).toUpperCase(),
```

Comment:
`getCurrencyDisplayCode` returns `toUpperCase` already, why do we need this?

Reply:
yeah, I think this was added by mistake in the commit.